berkdb/lock/lock_deadlock.c

```diff
@@ -1363,7 +1363,7 @@ retry:	count = region->stat.st_nlockers;
 				logmsg(LOGMSG_USER, "Marking valid waiter after increment lip %p id=%x dd_id=%x count=%u master=%x\n",
 					lockerp, lockerp->id, lockerp->dd_id,
 					dd_id_array[lockerp->dd_id].count,
-					(has_master)? lockerp->master_locker : 0);
+					(unsigned int)((has_master)? lockerp->master_locker : 0));
```

Comment:
`master_locker` is `intptr_t` -- lets use `PRIiPTR`/`PRIxPTR` to print instead of casting to `unsigned`

Reply:
I can try that.